libbeat/processors/add_kubernetes_metadata/kubernetes.go

```diff
@@ -316,19 +313,16 @@ func (k *kubernetesAnnotator) Run(event *beat.Event) (*beat.Event, error) {
 		return event, nil
 	}
 	if kubernetesMetadataExist(event) {
-		k.log.Debug("Skipping add_kubernetes_metadata processor as kubernetes metadata already exist")
 		return event, nil
 	}
+
 	index := k.matchers.MetadataIndex(event.Fields)
 	if index == "" {
-		k.log.Debug("No container match string, not adding kubernetes data")
 		return event, nil
 	}
 
-	k.log.Debugf("Using the following index key %s", index)
 	metadata := k.cache.get(index)
 	if metadata == nil {
-		k.log.Debugf("Index key %s did not match any of the cached resources", index)
```

Comment:
And this one is the second. I dont expect many occurencies to come from this

Reply:
I am unsure. Maybe in that case we should keep the printing line in `addPod` that tells us which indices were added? It should occur less.

Reply:
I dont like the idea then to have a message for every pod. I think we can live without the indices msg.

Reply:
Ok, so no debug message at all in the end?

Reply:
I would say yes. Lets wait for a second reviewer here as well and we can merge this